lib/tasks/argo_testing.rake

```diff
@@ -23,3 +32,56 @@ task(:default).clear
 
 desc 'run specs and rubocop (for CI)'
 task default: %i[rubocop spec]
+
+fedora_files = File.foreach(File.join(File.expand_path('../../fedora_conf/data', __dir__), 'load_order')).to_a
+
+namespace :argo do
+  namespace :repo do
+    desc "Load XML file(s) into repo (fedora and solr), default: contents of 'load_order' file. With a glob: rake argo:repo:load[fedora_conf/data/*.xml]"
+    task :load, [:glob] => :environment do |_task, args|
+      puts "travis_fold:start:argo-repo-load\r" if ENV['TRAVIS'] == 'true'
+
+      file_list = []
+      if args.key?(:glob)
+        file_list = Dir.glob(args[:glob])
+      else
+        puts 'No file glob was specified so file order and inclusion is determined by the load_order file'
+        file_list = load_order_files(fedora_files)
+      end
+
+      errors = []
+      i = 0
+
+      require 'webmock'
+      # only allow connections to Fcrepo, Solr and workflow
+      WebMock.disable_net_connect!(allow: ['workflow:3000',
+                                           'solr:8983',
+                                           'fcrepo:8080',
+                                           'localhost:8983',
+                                           'localhost:8984',
+                                           'localhost:3004',
+                                           'dor-indexing-app:3000'])
+      include WebMock::API
+      WebMock.enable!
+      file_list.each do |file|
+        i += 1
+
+        handler = proc do |e, _attempt_number, _total_delay|
+          puts warn "ERROR loading #{file}:\n#{e.message}\n#{e.backtrace.join "\n"}"
+          errors << file
+        end
+        pid = "druid:#{File.basename(file, '.xml')}"
+        with_retries(max_tries: 3, handler: handler, rescue: [StandardError]) do |attempt|
+          puts "** File #{i}, Try #{attempt} ** file: #{file}"
+
+          ActiveFedora::FixtureLoader.import_to_fedora(file, pid)
```

Comment:
I want to remove this dependency next week

Reply:
with_retries?

Reply:
Can we take it out now?

Reply:
ActiveFedora will be removed next week.  Maybe this can go in dor-services-app.